docs/make.jl

```diff
@@ -8,7 +8,7 @@ using ClimaLSM
 include("pages_helper.jl")
 tutorials = [
     "For model developers" => [
-        "Using AbstractModel functionality" => "Usage/model_tutorial.jl",
+        #        "Using AbstractModel functionality" => "Usage/model_tutorial.jl",
```

Comment:
I think we can remove the Henon Heiles tutorial in favor of more relevant tutorials, and just delete this instead of commenting it out (as discussed earlier)